tests/support/helpers.py

```diff
@@ -1656,3 +1657,35 @@ def dedent(text, linesep=os.linesep):
     if not isinstance(text, six.text_type):
         return salt.utils.stringutils.to_bytes(clean_text)
     return clean_text
+
+
+class CaptureSTDs(object):
```

Comment:
There *may* be a better name than `CaptureSTDs`. `CaptureOutput`? Maybe two decorators, a `capture_stdout` and a `capture_stderr`?

Reply:
The same class could be made to optionally capture each of the outputs.

:+1: on the name change

Reply:
FYI, pytest has this out of the box :)

Reply:
Addressed